### DISCUSSION TITLE
Hue dimmer switch v2 scene button to trigger zigbee scenes

### DISCUSSION THREAD

**Discussion by jonasjeeliasson** [Koenkk/zigbee2mqtt#8542] [General]
Hi!

I've just started out z2m and really happy with how well it works and perform. Have around 70 devices and everything is snappy and always works.

For all rooms I've grouped the  lights and bound them directly to my different hue dimmer v2 switches. This works great as well and all lights can be controlled even though z2m and/or HA is down. I've also created some zigbee scenes with the scene store command which I can recall with mqtt to trigger different scenes.

Now to the question, can I make the scene button on the hue dimmer v2 switch trigger these different scenes some how without being dependent on the coordinator or HA? Would have loved like: first click trigger scene 1, second click trigger scene 2 and so on.

**Comment by basistam** [comment]
I'm very interested in similar feature, recently I've replaced hue bridge with zigbee2mqtt integration and I dont miss a thing except for scene switches.

Would be great if we can assign button to loop through defined scenes for specific group.

**Reply by Bascht74** [reply]
Same question here...
After switching from hue bridge and digging into Z2M this is something that I am missing as well... Ideally I don't need to use HA automatisations for that...

**Comment by ahocquet** [comment]
Knowing it's already possible to change scenes with an [Ikea Tradfri remote](https://www.zigbee2mqtt.io/devices/E1524_E1810.html#binding), there must be a way doing the same thing with a Hue Dimmer Switch.

I'm also very interested if somebody knows a way to make it works.

**Comment by nvanggaard** [comment]
I'm looking for the exact same. I can see that i can bind scene from hue dimmer to a hue bulb, but unfortunately i cannot get it to work.
I've tried investigating the docker logs. If i press level up / level down, it outputs that the cluster genLevelCtrl is called, but i cannot see the scene cluster when i click on/off. 

Is there any way of doing this?

**Comment by eparment** [comment]
I also just ran into this issue. I'm relatively new to z2m, so far my hue dimmer remote is working great but I'm a little disappointed that the hue button is essentially useless now. Is there any workaround?

**Comment by ondrejbaranek** [comment]
Hue dimmer doesn't have buttons that you can dedicate for scenes like Ikea tradfro remote. You will probably need some automation for this (eg. in home assistant)

**Reply by nvanggaard** [reply]
How does it then work when using hue bridge ? 
With the bridge it is possible

**Comment by eparment** [comment]
Actually, I found a somewhat reasonable workaround. 
For people running into the same issue, check out this: [https://github.com/Koenkk/zigbee-herdsman-converters/pull/3815](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3815)
There seems to be a 'magic group number' that you can use.
